peer_mentoring/posts/models.py

```diff
@@ -15,5 +16,7 @@ class Post(models.Model):
     author = models.ForeignKey(User, on_delete=models.CASCADE)
     title = models.CharField(max_length=150)
     content = models.TextField(blank=False, null=True)
-    comments = models.ManyToManyField(Comments)
+    comments = models.ManyToManyField(Comment)
     added = models.DateTimeField(auto_now_add=True, null=False, blank=False)
+    is_group_post = models.BooleanField(blank=False, null=False, default=True)
```

Comment:
I would not make it optional so just `models.BooleanField(default=True)`

Reply:
could this be a property / derived attribute? e.g.

```
@property
def is_group_post(self):
    return self.group_id is not None
```

then you don't need the extra column

Reply:
I'll look into this, I haven't seen this.

Reply:
What about if we want to add it to personal profiles? Starting to think that this is becoming too robust and maybe focus on groups being the MVP with the forums and chat? If we set this as a property for groups, then could we set a second property? No right, because it seems that the property setting makes it specific to the group, because then we can use the setter property to change the internal value for group, am I correct in my interpretation?

Reply:
Not sure if I follow, let's discuss this today ...